src/components/course-listing.cjsx

```diff
@@ -60,12 +60,19 @@ CourseListing = React.createClass
       isStudent = _.find roles, (role) -> role.type is 'student'
       isTeacher = _.find roles, (role) -> role.type is 'teacher'
 
+      courseType = if isConceptCoach then 'Concept Coach' else 'Tutor'
+
+      if courselink?
+        courseTypeFlag = "(#{courseType})"
+      else
+        courseTypeFlag = <div className='course-type-flag'>{courseType}</div>
+
```

Comment:
Yeah I noticed the same thing.  If you reverse the two `if courselink?` conditions you can see it is being used for `alt-link` which seems to be just a text only version of the course list.  Not sure we need this; it seems `if courselink?` is never true until courselink is defined anyway, I was just matching what was there.  It may just be old code...